config/install.bat

```diff
@@ -0,0 +1,3 @@
+python -m venv env
+call env\Scripts\activate.bat
```

Comment:
what is the .bat for? is it useful for you on windows?

Reply:
Yes, it is what needed to install the env in window. I can remove it if you prefer

Reply:
It's cool.